README.md

```diff
@@ -75,8 +75,8 @@ yarn link `path/to/openmrs-form-engine-lib`
 For example, if you are working on the `openmrs-esm-form-builder` module, you would run the following command:
 
 ```bash
+pwd | pbcopy # copy the path to form-engine-lib to the to the clipboard
 cd `path/to/openmrs-esm-form-builder`
```

Comment:
@samuelmale  @denniskigen  @ibacher  I think this was backwards?  (note I didn't test, as I don't have pbcopy on my machine... I'm guessing that's a macOS command?)

Reply:
Yeah, looks backwards to me...